partcad/src/partcad/part_factory_enrich.py

```
@@ -19,50 +19,62 @@
+                    source_project_name = source_project.name
+
+            pc_logging.debug(f"Initializing an enrich to {source_project_name}:{source_part_name}")
+
+            super().__init__(ctx, source_project, target_project, config)
+            self.source_project = source_project
+            self.source_project_name = source_project_name
+
+            if ";" in source_part_name:
+                self.source_part_name = source_part_name.split(";")[0]
+                suffix = source_part_name.split(";")[1]
+                self.extra_with = list(map(lambda p: p.split("="), suffix.split(",")))
```

Ruff (0.8.2)

Unnecessary `map()` usage (rewrite using a list comprehension)

Replace `map()` with a list comprehension

(C417)